crates/biome_js_analyze/tests/specs/nursery/useAnchorHref/invalid.jsx

```diff
@@ -0,0 +1 @@
+<a>Link</a> 
```

Comment:
There's no need to have two files to test the rule. You can move this inside `invalid-2.jsx`

```jsx
<>
	<a>Link</a> 
	<a target="_blank">External</a> 
</>

```